src/components/CommonComponents/SearchBar/index.tsx

```diff
@@ -32,7 +32,7 @@ const MotionCloseIcon = motion(CloseIcon);
 
 const SearchBar = (): JSX.Element => {
   const [query, setQuery] = useState('');
-  const searchInputRef = createRef<HTMLInputElement>();
+  const searchInputRef = useRef<HTMLInputElement | null>(null);
```

Comment:
`searchInputRef` can be `null` so you should add a check on line `62` to if `searchInputRef` is defined.